trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_vortex_distribution.py

```diff
@@ -1062,7 +1062,7 @@ def generate_fuselage_and_nacelle_vortex_distribution(VD,fus,n_cw,n_sw,precision
             fhs.nose_length   = ((1 - ((abs(h_array[i]/semispan_h))**fus_nose_curvature ))**(1/fus_nose_curvature))*fus.lengths.nose
             fhs.tail_length   = ((1 - ((abs(h_array[i]/semispan_h))**fus_tail_curvature ))**(1/fus_tail_curvature))*fus.lengths.tail
             fhs.nose_origin   = fus.lengths.nose - fhs.nose_length
-            fhs.origin[i][:]  = np.array([origin[0] + fhs.nose_origin , origin[1] + h_array[i], origin[2]])
+            fhs.origin[i][:]  = np.array([fhs.nose_origin , h_array[i], 0.]) # Local origin
```

Comment:
as per this change, the origin variable isn't being used anymore right? It looks like fus.origin is used later on instead. If so, you can prob remove 'origin'

Reply:
wait nvm I see it's being used for fvs origin